weed/command/server.go

```diff
@@ -132,6 +132,7 @@ func init() {
 	serverOptions.v.idxFolder = cmdServer.Flag.String("volume.dir.idx", "", "directory to store .idx files")
 	serverOptions.v.inflightUploadDataTimeout = cmdServer.Flag.Duration("volume.inflightUploadDataTimeout", 60*time.Second, "inflight upload data wait timeout of volume servers")
 	serverOptions.v.hasSlowRead = cmdServer.Flag.Bool("volume.hasSlowRead", false, "if true, this prevents slow reads from blocking other requests, but large file read P99 latency will increase.")
+	serverOptions.v.readBufSize = cmdServer.Flag.Int("volume.readBufSize", 1024 * 1024, "larger values can optimize query performance but will increase some memory usage,Use with hasSlowRead normally")
```

Comment:
change to readBufferSize

add `<experimental>`

Reply:
We are testing that `clickhouse` is stored in `weedfs`. In the case of a large number of `put` and `delete`, this parameter is effective。

Reply:
@famosss you mean clickhouse s3-tables in seaweedfs?

Reply:
> @famosss you mean clickhouse s3-tables in seaweedfs?



> 



> @famosss you mean clickhouse s3-tables in seaweedfs?

We are testing clickhouse `mergetree` engine, `s3_only` and `hot-warm(s3)` storage_policy.

Reply:
this is worthy of a wiki page on how to adjust these parameters.